### ISSUE TITLE
September newsletter

### ISSUE THREAD

**Issue by KirstieJane** [alan-turing-institute/AutisticaCitizenScience#34]
We sent out our [first newsletter on Aug 5 2019](https://tinyletter.com/AutisticaTuringCitizenScience/letters/news-about-the-autistica-turing-citizen-science-project).

Lets aim to send out the next one in the first week of September 😄 

This issue is to collect together content that we want to include in that newsletter.

@GeorgiaHCA, can you draft one and open a PR when you're ready? We should also collect together the newsletters in this github repository. Lets make a folder called `newsletters` and put them in there.

**Comment by KirstieJane** [comment]
We've got a website now: https://alan-turing-institute.github.io/AutisticaCitizenScience

We've received feedback that there are too many ways to contribute so we're working on making it clearer. The easiest way to contribute is to add your thoughts (including questions and priorities) to the google form: https://bit.ly/AutisticaTuringCitSciForm

**Comment by GeorgiaHCA** [comment]
I've made a draft for the September newsletter, and tried to make it as clear as possible. I've also made a folder for newsletters (done as a separate pull request).

For the focus groups, I didn't include details or dates - this was partly to limit information overload, but also because people will be recruited from the Insight group, and I didn't want people to think we were recruiting via. the newsletter.

Thoughts appreciated?

G
